src/client/java/teammates/client/scripts/scalabilitytests/InstructorFeedbackResultsPageDataGenerator.java

```diff
@@ -191,21 +192,17 @@ private InstructorFeedbackResultsPageDataGenerator(int numQuestions, int numStud
     }
 
     public static void main(String[] args) throws IOException {
-        //Number of students and questions for each data set.
-        int[] studentNums = {10, 20};
-        int[] questionNums = {1, 5, 10};
+        // One set of maximum number of students and questions to create single data set for all tests
```

Comment:
`Maximum number of students and questions to create a single data set for all tests.`